common/app/views/fragments/header.scala.html

```diff
@@ -8,73 +8,91 @@
 to apply any necessary changes to non-shared code there too.
 -->*@
 
-<header id="header" class="l-header u-cf" role="banner" data-link-name="global navigation: header">
-    <div class="l-header-pre gs-container u-cf">
-        <div class="brand-bar">
-            @if(IdentityProfileNavigationSwitch.isSwitchedOn) {
-                <div class="brand-bar__item brand-bar__item--profile popup-container has-popup brand-bar__item--has-control js-profile-nav" data-component="identity-profile">
-                    <a href="@Configuration.id.url/signin" data-link-name="User profile" data-toggle="popup--profile"
-                       class="brand-bar__item--action popup__toggle" data-test-id="sign-in-link"
-                       aria-haspopup="true">
-                        @fragments.inlineSvg("profile-36", "icon", List("rounded-icon", "control__icon-wrapper"))
-                        <span class="js-profile-info control__info" data-test-id="sign-in-name">sign in</span>
-                    </a>
-                    <div class="js-profile-popup"></div>
-                    <div class="js-popup-signed-out popup popup--signed-out u-h">
-                        <a class="popup--signed-out__link" href="@Configuration.id.url/signin" data-link-name="Signed out link"></a>
-                        You are signed out of the Guardian. <span class="popup--signed-out__highlight">Sign In</span>
-                        <button class="js-popup-toggle u-button-reset popup__toggle" data-link-name="Signed out link | Close">
-                            @fragments.inlineSvg("cross", "icon", Seq("popup__toggle-icon"))
-                        </button>
-                    </div>
-                </div>
-            }
+<header id="header"
+        class="l-header u-cf @if(metaData.contentType == "Interactive") { l-header--is-slim }"
```

Comment:
This would be useful for Identity to adopt if so, rather than keep prodding that our header version needs to be kept update manually!